libsolidity/codegen/ContractCompiler.h

```diff
@@ -130,7 +130,10 @@ class ContractCompiler: private ASTConstVisitor
 	/// body itself if the last modifier was reached.
 	void appendModifierOrFunctionCode();
 
-	void appendStackVariableInitialisation(VariableDeclaration const& _variable);
+	/// Creates a stack slot for the given variable and assigns a default value.
+	/// If the default value is complex (needs memory allocation) and @a _provideDefaultValue
+	/// is false, this might be skipped.
+	void appendStackVariableInitialisation(VariableDeclaration const& _variable, bool _provideDefaultValue);
```

Comment:
Even with `_provideDefaultValue` there's still a default, it's just zero. How about this:
```suggestion
	void appendStackVariableInitialisation(VariableDeclaration const& _variable, bool _allocateMemory);
```
Alternatively (note: this flips the flag):
```suggestion
	void appendStackVariableInitialisation(VariableDeclaration const& _variable, bool _zeroInitializeMemoryReferences);
```